scripts/produce_integers.py

```diff
@@ -0,0 +1,44 @@
+
```

Comment:
I'm not sure where to put this script, either here or in the main repo, as part of the `rht-labs-ad482` package.
- Pros for leaving as it is: the students can easily call and inspect it: `python scripts/produce_integers.py`
- Cons for leaving it as it is: It feels a bit dirty to leave this script isolated here, when it's clear that it requires the `rht-labs-ad482` modules to work (kafka, workspace...)
- Pros for moving it to `rht-labs-ad482`: the script is closer to its dependencies, which is cleaner, and the right thing to do (i think)
- Cons for moving it to `rht-labs-ad482`. We will have to adjust the setup.cfg to expose this script as a "binary", so that students can execute it. With this, `rht-labs-ad482` is not only a grading library, but also provides support to the exercises.

Reply:
I think leaving it here is fine. I am just not sure about the folder name, but keeping it generic for now wont make harm IMO.

Reply:
+1 to leave the script here

Reply:
Ok then. I'll keep it there. We can move it in a subsequent iteration